lib/propcheck.ex

```diff
@@ -429,20 +429,17 @@ defmodule PropCheck do
               unquote(prop)
             rescue
               e in ExUnit.AssertionError ->
-                stacktrace = System.stacktrace
```

Comment:
Perhaps it is safer to call here `__STACKTRACE__` instead of a few lines later such that the stack properly defined by the caller of the exception?